README.md

```diff
@@ -20,9 +20,10 @@ The console is a more friendly `kubectl` in the form of a single page webapp.  I
 
 1. [node.js](https://nodejs.org/) >= 8 & [yarn](https://yarnpkg.com/en/docs/install) >= 1.3.2
 2. [go](https://golang.org/) >= 1.8 & [glide](https://glide.sh/) >= 0.12.0 (`go get github.com/Masterminds/glide`) & [glide-vc](https://github.com/sgotti/glide-vc)
-3. [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and a k8s cluster
-4. `jq` (for `contrib/environment.sh`)
-5. Google Chrome/Chromium >= 60 (needs --headless flag) for integration tests
+3. Add the project to your $GOPATH: `$GOPATH/src/github.com/openshift/console`
```

Comment:
This is a list of dependencies. Since this isn't actually a dependency, it should be part of the instructions just below.

Sorry if it seems like I'm picking nits. I just think it would be easy to miss here.